configurations/default/env.schema.json

```diff
@@ -0,0 +1,172 @@
+  "type": "object",
+  "properties": {
+    "AUTH0_API_CLIENT": {
+      "type": "string",
+      "default": "test-auth0-client-id",
+      "description": "API client id required to authenticate with Auth0."
+    },
+    "AUTH0_API_SECRET": {
+      "type": "string",
+      "default": "test-auth0-secret",
+      "description": "API secret id required to authenticate with Auth0."
+    },
+    "AUTH0_DOMAIN": {
+      "type": "string",
+      "default": "test.auth0.com",
+      "description": "Auth0 tenant URL."
+    },
+    "AWS_PROFILE": {
+      "type": "string",
+      "default": "default",
+      "description": "AWS credentials"
+    },
+    "AWS_API_SERVER": {
+      "type": "string",
+      "default": "aws-api-id.execute-api.us-east-1.amazonaws.com",
+      "description": "For generating the swagger document at runtime. Can be null, however that will prevent tools such as swagger-UI from submitting test requests to the API server."
+    },
+    "AWS_API_STAGE": {
+      "type": "string",
+      "default": "stage-name",
+      "description": "For generating the swagger document at runtime. Can be null, however that will prevent tools such as swagger-UI from submitting test requests to the API server."
+    },
+    "BUGSNAG_API_KEY": {
+      "type": "string",
+      "default": "bugsnag-api-key",
+      "description": "A valid Bugsnag authorization token."
+    },
+    "BUGSNAG_EVENT_REQUEST_JOB_DELAY_IN_MINUTES": {
+      "type": "integer",
+      "default": "5",
+      "description": "Bugsnag event request frequency."
+    },
+    "BUGSNAG_EVENT_JOB_DELAY_IN_MINUTES": {
+      "type": "integer",
+      "default": "1",
+      "description": "Bugsnag event job frequency."
+    },
+    "BUGSNAG_PROJECT_JOB_DELAY_IN_MINUTES": {
+      "type": "integer",
+      "default": "1",
+      "description": "Bugsnag project job frequency."
+    },
+    "BUGSNAG_ORGANIZATION": {
+      "type": "string",
+      "default": "bugsnag-organization-id",
+      "description": "A valid Bugsnag organization id."
+    },
+    "BUGSNAG_PROJECT_NOTIFIER_API_KEY": {
+      "type": "string",
+      "default": "bugsnag-project-api-key-id",
+      "description": "A valid Bugsnag project API key."
+    },
+    "BUGSNAG_REPORTING_WINDOW_IN_DAYS": {
+      "type": "integer",
+      "default": "14",
+      "description": "Specifies how far in the past events should be retrieved."
+    },
+    "DEFAULT_USAGE_PLAN_ID": {
+      "type": "string",
+      "default": "usage-plan-id",
+      "description": "AWS API gateway default usage plan used when creating API keys for API users."
+    },
+    "OTP_API_ROOT": {
+      "type": "string",
+      "default": "http://otp-server.example.com/otp",
+      "description": "The URL of an operational OTP server."
+    },
+    "OTP_PLAN_ENDPOINT": {
+      "type": "string",
+      "default": "/routers/default/plan",
+      "description": "The OTP server's endpoint for plan."
```

Comment:
"The path to the OTP server trip planning endpoint."

Reply:
Updated.